site/app/views/admin/UsersView.php

```diff
@@ -139,14 +139,15 @@ public function sectionsForm($students, $reg_sections, $not_null_counts, $null_c
         }
 
         $this->core->getOutput()->addInternalCss('rotatingsectionsform.css');
-
         return $this->core->getOutput()->renderTwigTemplate("admin/users/RotatingSectionsForm.twig", [
             "students" => $students,
             "reg_sections" => $reg_sections,
             "reg_sections_count" => $reg_sections_count,
             "not_null_counts" => $not_null_counts,
             "null_counts" => $null_counts,
             "max_section" => $max_section,
+            "is_self_register_course" => $this->core->getQueries()->isSelfRegistrationAllowed($this->core->getConfig()->getCourse()) > 0,
+            "default_section" => $this->core->getQueries()->getDefaultRegistrationSection($this->core->getConfig()->getTerm(), $this->core->getConfig()->getCourse()),
```

Comment:
It is poor practice to execute database queries in a view.  Instead, it would be better to perform these queries in the controller and pass the result into the view.